MdeModulePkg/Bus/Pci/PciBusDxe/PciDeviceSupport.c

```diff
@@ -15,6 +15,83 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 //
 LIST_ENTRY  mPciDevicePool;
 
+// MU_CHANGE Begin - Disable BME at Exit Boot Services
+
+/**
+ * DisableBmeOnTree
+ *
+ * @param Bridge

```

Comment:
This parameter really should be updated to match the actual parameter name with a description.

Reply:
Updated to include description of the function and parameter.